.github/workflows/publish-prod.yaml

```diff
@@ -32,6 +32,8 @@ jobs:
         run: npm run build
         env:
           NODE_OPTIONS: --max_old_space_size=8192
+          DOCS_SITE_URL: https://docs.camunda.io
+          DOCS_SITE_BASE_URL: /
```

Comment:
These are the default values in docusaurus.config.js, but making them explicit here allows for better find/replace in the archival process.